### DISCUSSION TITLE
FPGA Accelerator

### DISCUSSION THREAD

**Discussion by PythonLinks** [acoular/acoular#372] [Ideas]
I am interested in building audio applications on FPGAs.  
Both for data collection from Microphone arrays using PDM microphones, 
and for real-time audio camera displays. 
I am curious if this would be of interest to anyone, and how you would use it?

My qualifications:  MIT Physics degree, Master's in Electrical Engineering Digital Design, and started using Python last century.  I am located in central Europe.

**Comment by esarradj** [comment]
I think the most appropriate use of FPGAs in this context is the data acquisition from _many_ microphones (hundreds or even thousands). For data and results processing most advanced algorithms that we use are probably not a good fit for FPGAs, given the algorithmic complexity and the availability of fast gpu/cpu solutions.

One bold example of (ultra-cheap) FPGA based use with a microphone array is [here](https://github.com/kingoflolz/mic_gateware).

**Reply by PythonLinks** [reply]
Very helpful.  
Thank you.
